entry.go

```diff
@@ -32,7 +29,11 @@ func NewEntry(log *Logger) *Entry {
 }
 
 // WithFields returns a new entry with `fields` set.
-func (e *Entry) WithFields(fields Fielder) *Entry {
+func (e *Entry) WithFields(fields Fielder) Interface {
+	return e.withFields(fields)
+}
+
+func (e *Entry) withFields(fields Fielder) *Entry {
```

Comment:
Don't forget a comment :DDD

Reply:
ha will do (comment and move interface definitions).